models/users.js

```diff
@@ -186,6 +186,36 @@ const fetchUserImage = async (users) => {
   return images;
 };
 
+/**
+ *
+ *
+ * Fetch Users with given Skill
+ *
+ * @param skill { String }: Get users with given skill
+ * @return {Promise<skills|Array>}
+ */
+async function userWithSkill(skill) {
+  try {
+    const userData = await userModel.orderBy("skills", "asc").get();
+    const userSkills = userData.docs.map((doc) => doc.data());
```

Comment:
userSkills has not only user skills data but also users' other data,
What are your thoughts on renaming the variable?